src/targets/cpu/include/migraphx/cpu/pointwise.hpp

```diff
@@ -24,6 +24,8 @@
 #ifndef MIGRAPHX_GUARD_AMDMIGRAPHX_CPU_POINTWISE_HPP
 #define MIGRAPHX_GUARD_AMDMIGRAPHX_CPU_POINTWISE_HPP
 
+#include <array>
```

Comment:
Is this being used ?

Reply:
Yes, implicitly in the header file (at least on Windows). Without it the compilation fails - fatal error.